central/cve/image/datastore/datastore_impl.go

```diff
@@ -116,71 +112,53 @@ func (ds *datastoreImpl) Suppress(ctx context.Context, start *types.Timestamp, d
 		return err
 	}
 
-	cves, missing, err := ds.storage.GetMany(ctx, ids)
+	vulns, err := ds.searcher.SearchRawCVEs(ctx, pkgSearch.NewQueryBuilder().AddExactMatches(pkgSearch.CVE, cves...).ProtoQuery())
```

Comment:
Same here, let's just run a GetMany and if some are missing then some are missing?

Reply:
Or does is the CVE not the ID anymore...

Reply:
we did not have cve before. We have it now. ID always existed.